client/src/components/DataFiles/DataFilesListing/DataFilesListing.jsx

```diff
@@ -168,7 +168,7 @@ const DataFilesListing = ({ api, scheme, system, path, isPublic }) => {
     path
   ) {
     if (isAtHomeDir) {
-      return isRootDir ? 'Root' : systemDisplayName;
+      return systemDisplayName;
```

Comment:
For WP-420 search placeholder needs to change when going into a new project, was "Root" before. Needs to show current folder instead